app/controllers/projects_controller.rb

```diff
@@ -10,7 +10,9 @@ def index
   end
 
   def create
-    @project = Project.new project_params
+    pp = project_params
+    pp[:customers] = Customer.where(id: pp[:customers])
+    @project = Project.new pp
```

Comment:
```suggestion
    project_params[:customers] = Customer.where(id: pp[:customers])
    @project = Project.new project_params
```
để như vầy k được à hỏi thôi nhá

Reply:
không, tại cái kia là hàm á đổi xong nó ko lưu lại đâu